src/oscar/apps/dashboard/ranges/apps.py

```diff
@@ -27,6 +27,8 @@ def get_urls(self):
             path('<int:pk>/', self.update_view.as_view(), name='range-update'),
             path('<int:pk>/delete/', self.delete_view.as_view(), name='range-delete'),
             path('<int:pk>/products/', self.products_view.as_view(), name='range-products'),
+            path('<int:pk>/products/<int:active_tab>/', self.products_view.as_view(),
```

Comment:
I still don't think this is necessary. If you want to remember what the active tab was before submiting the form, I suggest using a new HiddenFormField which you initially set on the forms.